### PR TITLE
feat: 🍰 Rebranding And White-Labeling 

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
Have all the information for the brand in separate config files. Set these defaults to ocelot.social

### CODE REVIEW

**Review by Elweyn — COMMENTED** [review]
The changes seem good in general! Their was just one thing that I saw when I started the App, we don't have a favicon.ico for the ocelot.social projekt, their is still the Human-Connection one!
And the following questions

File: backend/src/middleware/email/templates/emailVerification.html [hunk 1]
```diff
@@ -75,8 +75,8 @@
             <p style="margin: 0;">Sollte der Button für Dich nicht funktionieren, kannst Du auch folgenden Code in
               Dein Browserfenster kopieren: <span style="color: #17b53e;">{{{ nonce }}}</span></p>
             <p style="margin: 0; margin-top: 10px;">Bis bald bei <a href="https://human-connection.org"
```
[the remark this answers is not in the document]
Comment (Mogge) [reply backend/src/middleware/email/templates/emailVerification.html:77]: Thanks for finding ...

File: backend/src/middleware/email/templates/layout.html [hunk 2]
```diff
@@ -170,8 +170,7 @@
           <td
             style="padding: 20px; font-family: Lato, sans-serif; font-size: 12px; line-height: 15px; text-align: center; color: #888888;">
             <br><br>
-            Human Connection gGmbH<br><span class="unstyle-auto-detected-links">Bahnhofstraße 11, 73235 Weilheim /
-              Teck<br>Germany</span>
+            {{ORGANIZATION_NAME}}
```
[the remark this answers is not in the document]
Comment (Mogge) [reply backend/src/middleware/email/templates/layout.html:173]: OK. I give a link to the web page of the organization.

File: webapp/locales/de.json [hunk 3]
```diff
@@ -793,51 +743,11 @@
     }
   },
   "termsAndConditions": {
-    "addition": {
```
[the remark this answers is not in the document]
Comment (Mogge) [reply webapp/locales/de.json:796]: We reduced them to a single html file. No translations needed,

File: webapp/Dockerfile [hunk 4]
```diff
@@ -1,5 +1,4 @@
 FROM node:12.19.0-alpine3.10 as base
-LABEL Description="Web Frontend of the Social Network Human-Connection.org" Vendor="Human-Connection gGmbH" Version="0.0.1" Maintainer="Human-Connection gGmbH (developer@human-connection.org)"
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply webapp/Dockerfile:2]: I would prefer this here:

```
LABEL Description="Web Frontend of the Social Network ocelot.social" Vendor="ocelot.social Community" Version="0.0.1" Maintainer="ocelot.social Community (devops@ocelot.social)"
```

File: webapp/Dockerfile.maintenance [hunk 5]
```diff
@@ -1,5 +1,4 @@
 FROM node:12.19.0-alpine3.10 as build
-LABEL Description="Maintenance page of the Social Network Human-Connection.org" Vendor="Human-Connection gGmbH" Version="0.0.1" Maintainer="Human-Connection gGmbH (developer@human-connection.org)"
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply webapp/Dockerfile.maintenance:2]: Would be nice to have this here:

```
LABEL Description="Maintenance page of the Social Network ocelot.social" Vendor="ocelot.social Community" Version="0.0.1" Maintainer="ocelot.social Community (devops@ocelot.social)"
```

File: backend/src/config/metadata.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+export default {
+  APPLICATION_NAME: 'ocelot.social',
+  APPLICATION_SHORT_NAME: 'ocelot',
+  APPLICATION_DESCRIPTION: 'Ocelot Social Network',
+  ORGANIZATION_NAME: 'busFaktor() e.V.',
+  ORGANIZATION_JURISDICTION: 'Augsburg',
```
Comment (Tirokk) [inline backend/src/config/metadata.js:6]:
```suggestion
  ORGANIZATION_JURISDICTION: 'City of Angles',
```

File: webapp/constants/metadata.js [hunk 7]
```diff
@@ -0,0 +1,7 @@
+export default {
+  APPLICATION_NAME: 'ocelot.social',
+  APPLICATION_SHORT_NAME: 'ocelot',
+  APPLICATION_DESCRIPTION: 'Ocelot Social Network',
```
Comment (Tirokk) [inline webapp/constants/metadata.js:4]:
```suggestion
  APPLICATION_DESCRIPTION: 'ocelot.social Community Network',
```

File: webapp/constants/metadata.js [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+  ORGANIZATION_NAME: 'buusFaktor() e.V.',
```
Comment (Tirokk) [inline webapp/constants/metadata.js:5]:
```suggestion
  ORGANIZATION_NAME: 'ocelot.social Community',
```

File: webapp/constants/metadata.js [hunk 9]
```diff
@@ -0,0 +1,7 @@
...
+  ORGANIZATION_JURISDICTION: 'Augsburg',
```
Comment (Tirokk) [inline webapp/constants/metadata.js:6]:
```suggestion
  ORGANIZATION_JURISDICTION: 'City of Angles',
```

File: webapp/locales/de.json [hunk 10]
```diff
@@ -76,54 +76,7 @@
     }
   },
   "code-of-conduct": {
-    "consequences": {
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply webapp/locales/de.json:79]: This should be configurable in the future somehow to be discussed.

Or the whole page could become a link to somewhere?
Or is this solved already different in the following files?

File: webapp/locales/en.json [hunk 11]
```diff
@@ -76,54 +76,7 @@
     }
   },
   "code-of-conduct": {
-    "consequences": {
```
Comment (Tirokk) [inline webapp/locales/en.json:79]: Same same 👆🏼

File: webapp/locales/es.json [hunk 12]
```diff
@@ -76,54 +76,7 @@
     }
   },
   "code-of-conduct": {
-    "consequences": {
```
Comment (Tirokk) [inline webapp/locales/es.json:79]: Same same 👆🏼

File: webapp/locales/it.json [hunk 13]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (Tirokk) [inline webapp/locales/it.json:1]: I guess this solves the problem we had with Italian @Mogge ?
Wonderful !!! 😍

**Review by Tirokk — CHANGES_REQUESTED** [review]

**Comment by Tirokk** [comment]
Hey @Mogge !
You are great !!! This is really a chunk of code. 🚀🚀💫💫💫
And brings us so much forward … YEAH !!!

Thanks that you did this big and huge work! 🤗😁

I had a look over it and have still found some suggestions or probably wrong things.
Sorry for that!

File: .github/workflows/ci.yml [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+
+      - name: Check translation files
+        run: |
+          scripts/translations/sort.sh
+          scripts/translations/missing-keys.sh
+
+      - name: Build neo4j image
+        uses: docker/build-push-action@v1.1.0
+        with:
+          repository: humanconnection/neo4j
```
[the remark this answers is not in the document]
Comment (Mogge) [reply .github/workflows/ci.yml:21]: I replaced it with ocelotsocialnetwork

[the remark this answers is not in the document]
Comment (Mogge) [reply .github/workflows/ci.yml:21]: Let us see what happens

File: deployment/helm/ocelot.social/values.yaml [hunk 15]
```diff
@@ -50,4 +50,4 @@ privateKeyPassphrase: "YTdkc2Y3OHNhZGc4N2FkODdzZmFnc2FkZzc4"
 mapboxToken: "cGsuZXlKMUlqb2lhSFZ0WVc0dFkyOXVibVZqZEdsdmJpSXNJbUVpT2lKamFqbDBjbkJ1Ykdvd2VUVmxNM1Z3WjJsek5UTnVkM1p0SW4wLktaOEtLOWw3MG9talhiRWtrYkhHc1E="
 uploadsStorage: "25Gi"
 neo4jStorage: "5Gi"
-developmentMailserverDomain: nitro-mailserver.human-connection.org
\ No newline at end of file
+developmentMailserverDomain: nitro-mailserver.human-connection.org
```
[the remark this answers is not in the document]
Comment (Mogge) [reply deployment/helm/ocelot.social/values.yaml:53]: There where some more in this file ...

File: docker-compose.override.yml [hunk 16]
```diff
@@ -1,23 +1,21 @@
 version: "3.4"
```
[the remark this answers is not in the document]
Comment (Mogge) [reply docker-compose.override.yml:1]: Looks like a deplyment for schools in motion for me.

[the remark this answers is not in the document]
Comment (Mogge) [reply docker-compose.override.yml:1]: For now I decide to not change it

File: neo4j/Dockerfile [hunk 17]
```diff
@@ -1,7 +1,4 @@
 FROM neo4j:3.5.14
-# community edition 👆🏼, because we have no enterprise licence 👇🏼 at the moment
-# FROM neo4j:3.5.14-enterprise
```
[the remark this answers is not in the document]
Comment (Mogge) [reply neo4j/Dockerfile:3]: OK. You known my opinion about comments.

File: backend/src/config/metadata.js (hunk 6 above)
Comment (Mogge) [reply backend/src/config/metadata.js:6]: By the seven hells. City of Angels then.

File: webapp/locales/en.json (hunk 11 above)
Comment (Mogge) [reply webapp/locales/en.json:79]: This is a static, localized html page now.
See `/webapp/locales/html/de/code-of-conduct.html`

File: webapp/locales/it.json (hunk 13 above)
Comment (Mogge) [reply webapp/locales/it.json:1]: ?

File: docker-compose.yml [hunk 18]
```diff
@@ -60,12 +61,10 @@ services:
     ports:
       - 7687:7687
       - 7474:7474
-    volumes:
-      - neo4j_data:/data
```
[the remark this answers is not in the document]
Comment (Tirokk) [reply docker-compose.yml:64]: Should be not deleted dear! 😘

**Review by Tirokk — CHANGES_REQUESTED** [review]
Hey @Mogge ,
we getting really close now. 🚀💫

There are still little things I recommend to change, but than that’s it I guess!
Lets see what you say … 🤗😃

File: webapp/Dockerfile.maintenance [hunk 19]
```diff
@@ -1,5 +1,5 @@
...
+ABEL Description="Maintenance page of the Social Network ocelot.social" Vendor="ocelot.social Community" Version="0.0.1" Maintainer="ocelot.social Community (devops@ocelot.social)"
```
[the remark this answers is not in the document]
Comment (Mogge) [reply webapp/Dockerfile.maintenance:2]: UPS

**Review by Tirokk — APPROVED** [review]
Hey @mogge and further on @roschaefer !!!
You really made a big step forward. ❤️🍀
New times are rising. 🤗🚀🚀🚀💫💫💫

We’ll rock it !!!

Storm trupper fucking

**Comment by Tirokk** [comment]
If I downgrade the node version in the Dockerfiles then the build is running @Mogge ! 🧐

**Pull request merged**